packages/flutter_test/lib/src/controller.dart

```diff
@@ -410,8 +414,18 @@ abstract class WidgetController {
   ///
   /// If you want the drag to end with a speed so that the gesture recognition
   /// system identifies the gesture as a fling, consider using [fling] instead.
-  Future<void> drag(Finder finder, Offset offset, {int pointer}) {
-    return dragFrom(getCenter(finder), offset, pointer: pointer);
+  ///
+  /// {@template flutter.flutter_test.drag}
+  /// By default, if the x or y component of offset is greater than [kTouchSlop], the
```

Comment:
When reading this doc I'm left with a few unanswered questions. Maybe you can add some more information to answer them for our curious users?

* Why are the gestures broken up into multiple gestures in the first place?
* Why should I ever modify touchSlopX or tochSlopY?
* If I change them how do I determine the values I want for them?